react-task/src/components/Input.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Input = ({ type, id, placeholder, helpText }) => {
+  return (
+    <div class="form-floating mb-3">
+      <input
+        type={type}
+        class="form-control"
+        id={id}
+        placeholder={placeholder}
+        aria-describedby={helpText && "help"}
+      ></input>
+      <label for="firstName">{placeholder}</label>
```

Comment:
Rather change the for to use the id field so its unique per Input

```suggestion
      <label for={id}>{placeholder}</label>
```

Reply:
This comment was missed